morebits.js

```diff
@@ -445,6 +445,9 @@ Morebits.quickForm.element.prototype.compute = function QuickFormElementCompute(
 					}
 				}
 			}
+			if (data.shiftClickSupport && data.type === 'checkbox') {
+				Morebits.checkboxShiftClickSupport(Morebits.quickForm.getElements(node, data.name));
```

Comment:
Subtle, but this implies the same logic from #908 that `current.name` implies it's a single checkbox.  Neat!